docs/project/changelog.md

```diff
@@ -33,6 +33,12 @@ substitutions:
   importing unvendored or removed stdlib modules fails.
   {pr}`2973`
 
+- {{ Breaking }} The default value of `fullStdLib` in {any}`loadPyodide` has been
+  changed to `false`. This means Pyodide now will not load distutils by default.
+  If `fullStdLib` is set to `true`, it will load all unvendored stdlib modules.
```

Comment:
Maybe add a note that it's preferable to load the required modules explicitly, instead of using `fullStdLib=true`.